wordpress-mobile/WordPress-Android
# Remove old about activity and related resources

Fixes #15803 

This PR removes the following unused files and updates their references since the new Unified About is already integrated into WPAndroid and Jetpack

- Removed AboutPreferenceCategory and related preferences from AppSettingsFragment
- deleted old AboutActivity and related resources
- deleted old LicencesActivity and related resources
- Clean up related strings and styles

To test:
- Launch the app
- On MySite, Go to Me by clicking on avatar on top right hand corner
- Click on App Settings, scroll down and notice there's no About the app section
- Return to Me by pressing back 
- Notice About WordPress (second from bottom) below Share WordPress with a friend, click on it to see new Unified About 


## Regression Notes
1. Potential unintended areas of impact
None as Unified About is already integrated

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Clean up unit tests

3. What automated tests I added (or what prevented me from doing so)
Not required

PR submission checklist:
